sdk/data/azcosmos/cosmos_global_endpoint_manager_test.go

```diff
@@ -23,6 +26,9 @@ func TestGlobalEndpointManager_GetWriteEndpoints(t *testing.T) {
 	verifier := headerPoliciesVerify{}
 	pl := azruntime.NewPipeline("azcosmostest", "v1.0.0", azruntime.PipelineOptions{PerCall: []policy.Policy{headerPolicy, &verifier}}, &policy.ClientOptions{Transport: srv})
 	req, err := azruntime.NewRequest(context.Background(), http.MethodGet, srv.URL())
+	if err != nil {
+		return
+	}
```

Comment:
Should these blocks in the tests actually just be `assert.NoError(t, err)`?  Otherwise, the tests will appear to pass if there is an error, right?